tests/fido_test.py

```diff
@@ -101,6 +101,57 @@ def test_fetch_headers(server_url):
     assert actual_headers.get('foo') == 'bar'
 
 
+def test_fetch_headers_keep_content_length_if_no_body(server_url):
+    headers = {'Content-Length': '0'}
```

Comment:
In all these tests, is the content-length value really a string (as opposed to an integer) ?